src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java

```diff
@@ -253,11 +253,28 @@ public void copyFeedbackSession(String feedbackSessionName, String courseId) {
         
         selectDropdownByVisibleValue(copiedCourseIdDropdown, courseId);
         
-        clickCopyTableAtRow(0);
+        clickCopyTableRadioButtonAtRow(3);
```

Comment:
It doesn't seem like we need this change anymore since the test added in the ui test is comprehensive and adequate. This might cause unintended changes in the testing.